pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerImpl.java

```diff
@@ -740,16 +740,17 @@ public void connectionOpened(final ClientCnx cnx) {
         }
 
         boolean isDurable = subscriptionMode == SubscriptionMode.Durable;
-        MessageIdData startMessageIdData = null;
```

Comment:
Why don't we just make this line `final MessageIdData startMessageIdData;`? That would limit the number of changes here.

Reply:
I realized that I was mistaken here. Please ignore this question.